css/03-generic.css

```diff
@@ -93,10 +93,18 @@ h2:before, .entry .entry-title:before {
 }
 
 .entry .entry-content > h2 {
-	margin-left: -1rem;
+	margin-left: -2rem;
```

Comment:
Was this tested on phones? Need to make sure the hardcoded values are still okay for smaller screens.

Reply:
Assuming you're referring to the two lines below

	margin-top: 64px;
	margin-bottom: 48px;

Just switched them to _rem_ values

However, the margin-left thing could get a little goofy on mobile. We may want to have those negative values change to zero values when there's no left margin to drift into...

Reply:
Rem values will have the same problem - they are still static sizes. You could do some fun calc with `vw` units (vw = viewport width / 1vw = 1% of the width of the viewport). Maybe start with `margin-top: calc( 2vw + 30px )`? Then resize the browser window to see how it responds.

Reply:
Oops - I requested your review too early. Will aim to get on this sometime today.

Reply:
@slambert Okay, send me a message when ready!

Reply:
It's ready @laras126